sonic_platform_base/sonic_xcvr/api/xcvr_api.py

```diff
@@ -133,6 +133,23 @@ def get_transceiver_threshold_info(self):
         """
         raise NotImplementedError
 
+    def get_transceiver_status(self):
+        """
+        Retrieves transceiver status of this SFP
```

Comment:
can you provide more details in the API description what does status mean

Reply:
sure, added.

Reply:
we have an existing API "get_transceiver_bulk_status",  should we consider combining the new function with this existing one? Otherwise, I would suggest reconsidering the name of the new API for differentiation.